eva/udfs/abstract/pytorch_abstract_udf.py

```diff
@@ -100,18 +101,32 @@ class PytorchAbstractTransformationUDF(AbstractTransformationUDF, Compose):
     Use PyTorch torchvision transforms as EVA transforms.
     """
 
-    def __init__(self, transforms):
-        Compose.__init__(self, transforms)
+    def __init__(self):
+        blur = tv_transforms.GaussianBlur(kernel_size=3, sigma=1)
+        Compose.__init__(self, [tv_transforms.ToPILImage(), blur])
 
-    def transform(self, frames: ArrayLike) -> ArrayLike:
-        return Compose.__call__(self, frames)
+    def transform(self, images: np.ndarray):
+        blur = tv_transforms.GaussianBlur(kernel_size=3, sigma=1)
```

Comment:
How should this be done correctly?

Reply:
Do you want to apply this transformation to all UDFs based on pytorch UDFs or it is option UDF can choose to apply or not? BTW, @gaurav274 the new [UDF design](https://github.com/georgia-tech-db/eva/blob/b8cfc51259cadc68b184b4ae2ef6da077cdcde6f/eva/udfs/abstract/pytorch_abstract_udf.py#L98) is not really used in EVA now?